modules/navier_stokes/test/tests/finite_element/ins/hdg/lid.i

```diff
@@ -0,0 +1,89 @@
+mu=4e-3
+
+[GlobalParams]
+  variable = face_vel_x
+  u = vel_x
+  v = vel_y
+  grad_u = grad_vel_x
+  grad_v = grad_vel_y
+  face_u = face_vel_x
+  face_v = face_vel_y
+  pressure = p
+  enclosure_lm = lm
+  mu = ${mu}
+  rho = 1
+[]
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = -1
+    xmax = 1
+    ymin = -1
+    ymax = 1
+    nx = 16
+    ny = 16
+    elem_type = TRI6
+  []
+[]
+
+[Variables]
+  [face_vel_x]
+    family = SIDE_HIERARCHIC
+  []
+  [face_vel_y]
+    family = SIDE_HIERARCHIC
+  []
+  [p]
+    family = L2_LAGRANGE
+  []
+  [lm]
+    family = SCALAR
+  []
+[]
+
+[AuxVariables]
+  [vel_x]
+    family = L2_LAGRANGE
+  []
+  [vel_y]
+    family = L2_LAGRANGE
+  []
+  [grad_vel_x]
+    family = L2_LAGRANGE_VEC
+  []
+  [grad_vel_y]
+    family = L2_LAGRANGE_VEC
+  []
+[]
+
+[HDGKernels]
+  [ns]
+    type = NavierStokesHDGKernel
+  []
+[]
+
+[HDGBCs]
+  [walls]
+    type = NavierStokesHDGVelocityDirichletBC
+    boundary = 'left right bottom'
+  []
+  [lid]
+    type = NavierStokesHDGVelocityDirichletBC
+    boundary = 'top'
+    dirichlet_u = '1'
+  []
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = NEWTON
+  petsc_options_iname = '-pc_type -pc_factor_shift_type'
+  petsc_options_value = 'lu       NONZERO'
```

Comment:
One of the reasonswhy HDG might be so good is that you can do field split preconditioning with it (if I remember correctly). I think it would be beneficial to the users to see an example with that. Unless there is already one, in which case feel free to resolve.

Reply:
HDG is no more amenable to field split than other finite element discretizations. It's draw is that it gives the same benefits of a DG (or FV) discretization in terms of local element conservation and natural ability to upwind convection schemes while allowing you to condense out the internal degrees of freedom that make a DG scheme have substantially more global degrees of freedom than a Continuous Galerkin discretization.

In at least one way HDG is *harder* to field split than CG. It takes a larger gamma coefficient in grad-div stabilization in order to make the Schur complement spectrally equivalent to the pressure mass matrix. This larger perturbation makes the momentum equation have worse conditioning. This is what I am currently researching. You can see https://gitlab.com/petsc/petsc/-/merge_requests/7550 for some work in that arena. 

All that is to say that field split is still a fair amount of work. It actually requires *not* condensing out the internal dofs, at least not until you are working on the block velocity level. I do have a field split example in https://github.com/idaholab/moose/pull/27188.

Reply:
Great, sounds good to me!